website/docs/tutorial/1a-setting-up-bigquery.md

```diff
@@ -0,0 +1,225 @@
+
+2. Generate credentials with the following options:
+
+    1. "Credential Type"
+
+        1. "Which API are you using?" `BigQuery API`
+
+        2. "What data will you be accessing?" `Application data` (you'll be creating a service account)
+
+        3. "Are you planning to use this API with App Engine or Compute Engine?" `No`
+            <Lightbox src="/img/bigquery/api-credential-type.png" title="Bigquery API - Credential Type" />
+
+        4. Click “NEXT”
+
+    2. “Service account details”
+
+        1. "Service account name" `dbt-user` (although I chose `dbt-learn-admin`)
+            > Note: You can enter a description for future reference, but this is optional.
+
+            <Lightbox src="/img/bigquery/api-service-account-details.png" title="Bigquery API - Service Account Details" />
+
+        2. Click “CREATE AND CONTINUE”
+
+    3. Create and/or grant access to project. 
+        > Note: The simplest way is to set up one owner/admin role, but you can set up different permissions here for developers or read-only if required.
+
+        <Lightbox src="/img/bigquery/api-grant-service-access.png" title="Bigquery API - Grant Service Account Access" />
+
+        1. then click “CONTINUE”
+
+    4. Grant users access:
+
+        1. Add your own email initially  
+            <Lightbox src="/img/bigquery/api-grant-user-access.png" title="Bigquery API - Grant Users Access" />
+
+        2. Click “DONE”
+
+3. You now have a service account. Click on it to proceed:
+    <Lightbox src="/img/bigquery/api-select-service-account.png" title="Bigquery API - Select Service Account" />
+
+    1. Under “KEYS” heading, click “ADD KEY” then “Create new key”
+        <Lightbox src="/img/bigquery/api-create-key.png" title="Bigquery API - Create New Key" />
+
+    2. "Key type" `JSON`
+        <Lightbox src="/img/bigquery/api-key-type.png" title="Bigquery API - Key Type" />
+
+    3. Click “CREATE” and it will download the keyfile
+
+## Set up a dbt Cloud project and connect to BigQuery
+
+<LoomVideo id="c75212052038463b977d0eb856d25ff1" />
+
+Above we created a project in Bigquery and created an API key so we interact with BigQuery from other applications. Now we are going to navigate to dbt Cloud and see if we can access our BigQuery Project!
+
+1. It's time to leave BigQuery and navigate to https://cloud.getdbt.com/ so we can create a new project
+
+2. Click the Hamburger
+
+3. Click "Account Settings"
+    
+    1. You should be in the "Projects" section by default. Click "New Project”, in the top right.
+        <Lightbox src="/img/bigquery/set-up-new-project.png" title="dbt Cloud - Set Up A New Project" />
+    
+    2. Click “Begin”
+
+4. Project Settings
+    1. Name your project. I chose 'BigQuery Demo', but you might consider your company's name or 'Arrakis' if you semi-haphazardly name all your projects and repos after Dune references. Click “Continue”.
```

Comment:
lol - I find this hilarious and would keep it haha. Yield to @runleonarun to check if this is cool as a fun little note.

Reply:
haha yeah we used to do that at WP. it was a mess.

Reply:
at WP you would name everything after Dune references? that's amazing

Reply:
@coapacetic and @patkearns10 I am 100% onboard! One of my favorite things is to plant little Easter eggs in docs (not necessarily Dune- though I love that).